openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -58,11 +61,10 @@ OpenAssessment.ResponseView.prototype = {
                 view.server.renderLatex($(stepID, view.element));
                 view.installHandlers();
                 view.setAutoSaveEnabled(true);
-
                 view.isRendering = false;
                 view.baseView.announceStatusChangeToSRandFocus(stepID, usageID, false, view, focusID);
                 view.announceStatus = false;
-
+                view.dateFactory.report();
             }
         ).fail(function() {
```

Comment:
This call is a bit redundant, but there's no harm done in calling the factory multiple times. It seemed, observationally, to increase the reliability of the functions.